injectproxy/routes.go

```diff
@@ -256,24 +257,48 @@ func (r *routes) enforceLabel(h http.HandlerFunc) http.Handler {
 	})
 }
 
-// getLabelValue returns the statically set label value, or the label value
-// sent through a URL parameter.
-// It returns an error when either the value is found in both places, or is not found at all.
+// getLabelValue returns a regular expression concatenated using "|" by the statically set label value,
+// or by the label value sent through a URL parameter.
+// It returns an error when the URL parameter cannot be parsed,
+// or the value is found in both places, or is not found at all,
+// or one of the provided parameters is empty
 func (r *routes) getLabelValue(req *http.Request) (string, error) {
```

Comment:
At this stage, we should return a slice of strings because not all proxied endpoints work with regexp matchers (e.g. /api/v1/rules)

```suggestion
func (r *routes) getLabelValues(req *http.Request) ([]string, error) {
```

Reply:
I have updated the code. The slice of label values are stored in the context. And in `query()`, `matcher()` and `passthrough()` the context will be processed in different way.